src/test/java/com/wikia/webdriver/testcases/articlecrudtests/ArticleFeaturesCRUDTestsAnon.java

```diff
@@ -2,6 +2,7 @@
 
 import com.wikia.webdriver.common.core.annotations.RelatedIssue;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'com.wikia.webdriver.common.core.annotations.RelatedIssue'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AUselessImportCheck)